src/ziggurat/init.clj

```diff
@@ -44,6 +46,10 @@
   (start-rabbitmq-connection args)
   (messaging-producer/make-queues (event-routes args)))
 
+(defn- set-properties-for-structured-logging []
+  (if (= (:log-format (ziggurat-config)) "json")
```

Comment:
Here, shouldn't we read from the environment variable ZIGGURAT_LOG_FORMAT? So that we can put the call to `set-properties-for-structured-logging` as the very first step in init.